src/comparable.ml

```diff
@@ -0,0 +1,13 @@
+module type S = sig
+  type t
+
+  include Equatable.S with type t := t
+
+  val compare:t Comparator.t
+end
+
+module Default(T: sig type t end) : S = struct
```

Comment:
Another data point, in order to do a fair performance comparison of Immutable.re's IntSet vs. an ocaml Set of ints, I had to do the following:

https://github.com/facebookincubator/immutable-re/blob/master/immutable/perf/SetPerfTester.re#L60-L73

Using polymorphic compare in CamlIntSet was giving IntSet a significant unfair advantage.